src/Web/Asset/BundleBuilder.php

```
@@ -59,7 +63,19 @@ final class BundleBuilder
     /**
      * Base URL of asset files.
      */
-    public private(set) string $baseUrl = '';
+    public private(set) string $baseUrl = '@baseUrl' {
```

PHPStan (2.0.3)

Multiple access type modifiers are not allowed on line 66


(phpstan.parse)